### PR TITLE
🔥destroy awesome type script loader🔥 #patch

### PR DESCRIPTION
**Pull request by GiantRobots**
## Merge Checklist

- [x] Has descriptive PR title and body
- [x] All changes are backwards compatible
- [ ] Code reviewed by a SendGrid frontend lead

### CODE REVIEW

File: .storybook/webpack.config.js [hunk 1]
```diff
@@ -66,6 +55,27 @@ module.exports = function(config, env) {
...
   });
 
+  config.module.rules.push({
+    exclude: [
+      /\.html$/,
+      /\.(js|jsx|ts|tsx)$/,
+      /\.css$/,
+      /\.scss$/,
+      /\.json$/,
+      // /\.svg$/,
```
Comment (stevekinney) [inline .storybook/webpack.config.js]: Should we just remove this line?

Comment (GiantRobots) [reply .storybook/webpack.config.js]: Prolly, it was definitely copy pasta. Any ideas on how to test this to make sure we didn't break everything ahead of time?

File: .storybook/preview-head.html [hunk 2]
```diff
@@ -1,30 +1,40 @@
-
 <style>
   body {
     padding: 4em !important;
-    margin:0px;
+    margin: 0px;
```
Comment (alexabreu) [inline .storybook/preview-head.html]: nit pick: margin: 0;

**Comment by thinkingserious** [comment]
Hello @GiantRobots,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**